client/question/services/FeedbackGeneratorService.js

```diff
@@ -519,11 +631,12 @@ tie.factory('FeedbackGeneratorService', [
             var testCases = testSuites[j].getTestCases();
             for (var k = 0; k < testCases.length; k++) {
               var testCase = testCases[k];
+              var testCaseIndex = k;
```

Comment:
You could just replace k with testCaseIndex in the for loop definition in line 632 -- in general, we try to avoid having two different names/variables for the same thing.

Reply:
Changed.